src/webapp/components/survey/hook/useSurveyForm.ts

```diff
@@ -151,6 +156,21 @@ export function useSurveyForm(formType: SURVEY_FORM_TYPES, eventId: string | und
         currentCaseReportForm?.id,
     ]);
 
+    useEffect(() => {
+        if (prevalenceCurrentFormId) {
```

Comment:
this is business logic and does not belong in presentation layer.. The question visibility should be set to false in domain layer